exchanges/bybit/bybit_test.go

```diff
@@ -496,6 +496,24 @@ func TestWSOrderExecution(t *testing.T) {
 	if err != nil {
 		t.Fatal(err)
 	}
+
+	pressXToJSON = []byte(`[{"e":"executionReport","E":"1688442177725","s":"USDTEUR","c":"1688442177507","S":"SELL","o":"MARKET_OF_BASE","f":"GTC","q":"2","p":"0","X":"NEW","i":"1457153032031615232","M":"0","l":"0","z":"0","L":"0","n":"0","N":"","u":true,"w":true,"m":false,"O":"1688442177715","Z":"0","A":"0","C":false,"v":"0","d":"NO_LIQ"}]`)
```

Comment:
Could you please enhance this to include some basic unit tests for things like order type, order side, timestamps, pairs etc?